pkgs/development/web/deno/update/librusty_v8.ts

```diff
@@ -74,7 +68,10 @@ export async function updateLibrustyV8(
 ) {
   log("Starting librusty_v8 update");
   // 0.0.0
-  const version = await getLibrustyV8Version(owner, repo, denoVersion);
+  const cargoLockData = await getCargoLock(owner, repo, denoVersion);
+  console.log(cargoLockData);
+  const packageItem = cargoLockData.package.find(({ name }) => name === "v8");
```

Comment:
Would this resolve https://github.com/NixOS/nixpkgs/pull/256753?

Reply:
Yes it would. Sorry I missed this @zvxzvxzv and thanks for the PR